packages/cli/internal/pkg/cli/context_describe.go

```diff
@@ -28,7 +30,17 @@ func newDescribeContextOpts(vars describeContextVars) (*describeContextOpts, err
 	}, nil
 }
 
-func (o *describeContextOpts) Validate() error {
+func (o *describeContextOpts) Validate(args []string) error {
+	if len(args) == 0 && o.ContextName == "" {
+		return fmt.Errorf("a context must be provided")
+	} else if len(args) == 1 && o.ContextName != "" {
+		return fmt.Errorf("either the '-c' flag or a contexts must be provided, but not both")
```

Comment:
`or a contexts` -> `or a list of contexts` or `or a context`?

Reply:
Good catch